public/app/core/components/RolePickerDrawer/RolePickerSelect.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+
+import { MultiSelect } from '@grafana/ui';
+
+import { RolePickerDrawer } from './RolePickerDrawer';
+
+export interface Props {}
+
+export const RolePickerSelect = ({}: Props): JSX.Element => {
```

Comment:
```suggestion
export const RolePickerSelect = ({}: Props) => {
```
Return type is better omitted when possible.

Reply:
I don't remember adding these values. I must have a pre-commit hook that changes them. I'll keep an eye on those. Thank you!